src/items/itemsTable.component.tsx

```diff
@@ -81,11 +84,15 @@ export function ItemsTable(props: ItemTableProps) {
     undefined
   );
 
+  const { data: systemTypesData, isLoading: systemTypesLoading } =
+    useGetSystemTypes();
+
+  const { data: usageStatusData } = useGetUsageStatuses();
   const systemIdSet = new Set<string>(
     itemsData?.map((item) => item.system_id) ?? []
   );
 
-  let isLoading = isLoadingItems;
+  let isLoading = isLoadingItems || systemTypesLoading;
```

Comment:
```suggestion
  let isLoading = isLoadingItems || isLoadingSystemTypes;
```
Would match the existing naming.